var/spack/repos/builtin/packages/vtk-m/package.py

```diff
@@ -158,6 +158,10 @@ class VtkM(CMakePackage, CudaPackage, ROCmPackage):
     # https://gitlab.kitware.com/vtk/vtk-m/-/merge_requests/3258
     patch("mr3258-fix-typo-thrust-dependency-with-rocm.patch", when="@2.2:")
 
+    # VTK-M PR#3259
+    # https://gitlab.kitware.com/vtk/vtk-m/-/merge_requests/3259
+    patch("mr3259-thrust-is_arithmetic-fix.patch", when="@2.2.0 +cuda ^cuda@12.6:")
```

Comment:
I suggest also applying the patch for VTK-m @2.1.0 and @2.0.0. The same patch file should apply to all three versions. If we supply a patch release for any of these, it will contain this patch.

Reply:
Will do!